include/mbedtls/mbedtls_config.h

```diff
@@ -1650,6 +1650,18 @@
 */
 #define MBEDTLS_SSL_EARLY_DATA
 
+/**
+ * \def MBEDTLS_SSL_MAX_EARLY_DATA_SIZE
+ *
+ * The maximium amount of 0-RTT data. It only work when MBEDTLS_SSL_EARLY_DATA
```

Comment:
```suggestion
 * The maximium amount of early data. It only work when MBEDTLS_SSL_EARLY_DATA
```

Reply:
That description is come from RFC8446 section 4.2.10 . I don't think we should change it.

Reply:
Please see comment https://github.com/Mbed-TLS/mbedtls/pull/6538#discussion_r1027913516.

Reply:
I will change that. That is come from RFC8446 section 4.6.1.  I do not think replace `0-RTT data` with `early data` is good

Reply:
> Please check latest code. the sentence is from RFC 8446. I do not think it should be change.
> 
What  latest code? can you paste it here?

Reply:
latest `development` , tls13-supported.md

Reply:
Any new feedback?